internal/handlers/handlers.go

```diff
@@ -33,6 +33,7 @@ var (
 	strPostGetOptions                         = []byte("POST, GET, OPTIONS")
 	strSlashBatch                             = []byte("/batch")
 	strSlashDebug                             = []byte("/debug")
+	strSlashHealth                            = []byte("/health")
```

Comment:
Ideally we should use /ping, since that's what the pro.ip-api.com endpoint uses as well.

Reply:
Works for me